### DISCUSSION TITLE
The example pet_sla_and_adt.py only one eddy_contour

### DISCUSSION THREAD

**Discussion by ddd0203** [AntSimi/py-eddy-tracker#246] [Q&A] [answered]
Hello,
When I run the pet_sla and adt.py in the examples, I can only get one eddy contour. The a_sla.effective_contour_ longitude is also a 1 * 50 ndarray. What is the reason for this? How can I solve this problem?
<img width="1122" alt="53de009f8c8bd470cb5843cdf47a0ee" src="https://github.com/user-attachments/assets/444c90b0-4d20-44e4-b19e-6ccae1ed8ce2">

**Comment by AntSimi** [comment, marked as answer]
Take a look at #217, check matplotlib version

**Reply by ddd0203** [reply]
ok,thank you very much!
